### ISSUE TITLE
Default Brand pages should include an 'Accessibility' page

### ISSUE THREAD

**Issue by boonebgorges** [cuny-academic-commons/commons-in-a-box#445]
See https://cboxopenlab.org/groups/help-forum/forum/topic/openlab-theme/#post-311

It would be nice for this page to include some language that guides folks toward some policy language, etc. Ping @bree-z, @jennaspevack

**Comment by bree-z** [comment]
So sorry for the delay! We'll provide the text after we meet on Thursday.

**Comment by bree-z** [comment]
Hi Boone - Below the text for the page. Thanks!

--

This page can be used to provide information about accessibility. We have provided a template that you can modify to meet the needs of your community (for instance, you may prefer to link to instructional materials created by your institution). Questions about accessibility on Commons In A Box OpenLab? Join the [discussion on the Community Hub](https://cboxopenlab.org/groups/the-hub/forum/). 

If you are the administrator, visit Dashboard > Pages to modify this text. 

## What is accessibility?

Accessibility means ensuring that people with hearing, visual, motor, neurological, cognitive, and other disabilities can use and interact with websites, tools, and technologies and the materials they contain. 

This is important because everyone should have equal access to information, regardless of ability; in many countries, compliance with accessibility standards is required by law. Making websites and materials accessible also often makes them easier for everyone to use and understand. 

Want to learn more? The [World Wide Web Consortium (W3C)](https://www.w3.org/)’s [Web Accessibility Initiative](https://www.w3.org/WAI/) provides a guide to [web accessibility fundamentals](https://www.w3.org/WAI/fundamentals/), including a helpful [introduction to accessibility](https://www.w3.org/WAI/fundamentals/accessibility-intro/). 

## Making this site accessible
This site is powered by Commons In A Box OpenLab, which is [designed to comply with the latest web accessibility standards](https://commonsinabox.org/faqs/is-cbox-openlab-accessible-for-people-with-disabilities). Commons In A Box is built using WordPress and other software created by the WordPress community; see[ WordPress’s accessibility statement](https://wordpress.org/about/accessibility/) for more information.

The administrators of this site also work to ensure that any features and functionality added to the site (using WordPress plugins, themes, and custom code) also comply with accessibility standards. 

## Making your work accessible
Members of this site are also responsible for ensuring that their work is accessible. Before you share your work, take a few moments to make sure it is accessible to everyone:  

- [Make media accessible](https://openlab.citytech.cuny.edu/accessibilitymodule/best-practices/accessible-media/) by adding descriptions (alt-text) to images and enabling captions for audio and video.
- Check that your [text and documents](https://openlab.citytech.cuny.edu/accessibilitymodule/best-practices/text-and-documents/) are easy to read and comply with accessibility standards.
- [Organize](https://openlab.citytech.cuny.edu/accessibilitymodule/best-practices/accessible-organization-and-layout/) your materials to make them easy to follow. 

## Questions?
If you have any questions or concerns about accessibility on this site, contact us at [administrators: provide contact information here].

**Credits**: This page uses content from the City Tech Library’s “[Introduction to Accessibility](https://openlab.citytech.cuny.edu/accessibilitymodule/),” the SUNY Oneonta OpenLab’s “[Accessibility in OpenLab](https://openlab.oneonta.edu/about/accessibility-in-openlab/),” and the W3C Web Accessibility Initiative’s introduction to [web accessibility fundamentals](https://www.w3.org/WAI/fundamentals/).

**Comment by boonebgorges** [comment]
Sorry, this missed the release deadline and will have to wait until the next release.

**Comment by boonebgorges** [comment]
I've added this feature. Here's the page on openlabdev.commonsinabox.org: https://openlabdev.commonsinabox.org/about/accessibility/

I added it as a child of 'About', so that it goes into the 'About' nav menu.

I also added an upgrade routine, so that existing sites will get the page.

**Comment by boonebgorges** [comment]
I've made the change. To test, you'll need to do a fresh installation/upgrade.

**Comment by bree-z** [comment]
Thanks Boone. As with #490 we'll close without testing since it's just a text change.
